DigitalLearningSolutions.Data.Tests/Helpers/CustomPromptsTestHelper.cs

```diff
@@ -15,26 +16,98 @@ public static CentreCustomPrompts GetDefaultCentreCustomPrompts
             CustomPrompt? customPrompt6 = null
```

Comment:
I feel like this should accept a list of CustomPrompt. Then when called we could do something like:
var customPrompts = new List<CustomPrompt>() { expectedPrompt1, expectedPrompt2 };
GetDefaultCentreCustomPrompts(centreId, customPropmts)